src/frozen_fields/main.py

```diff
@@ -163,45 +94,37 @@ def frozenToggle(self, batch=False):
             except IndexError:
                 break
 
-    if anki21:
-        self.loadNoteKeepingFocus()
-    else:
-        self.web.eval("saveField('key');")
-        self.loadNote()
+    self.loadNoteKeepingFocus()
 
 
-def onFrozenToggle21(self, batch=False):
+def onFrozenToggle(self, batch=False):
     self.web.evalWithCallback(
         "saveField('key');", lambda _: self.frozenToggle(batch=batch))
 
 
-def onSetupButtons20(self):
-    """Set up hotkeys"""
-    if not isinstance(self.parentWindow, AddCards):  # only modify AddCards Editor
-        return
-
-    QShortcut(QKeySequence(hotkey_toggle_field), self.parentWindow,
-              activated=self.frozenToggle)
-    QShortcut(QKeySequence(hotkey_toggle_all), self.parentWindow,
-              activated=lambda: self.frozenToggle(batch=True))
-
-
-def onSetupShortcuts21(cuts, self):
+def onSetupShortcuts(cuts, self):
     cuts += [(hotkey_toggle_field, self.onFrozenToggle),
              (hotkey_toggle_all, lambda: self.onFrozenToggle(batch=True), True)]
     # third value: enable shortcut even when no field selected
 
 # Add-on hooks, etc.
 
 
-if anki21:
-    addHook("setupEditorShortcuts", onSetupShortcuts21)
-    Editor.onBridgeCmd = wrap(Editor.onBridgeCmd, onBridge, "around")
-    Editor.loadNote = loadNote21
-    Editor.onFrozenToggle = onFrozenToggle21
-else:
-    addHook("setupEditorButtons", onSetupButtons20)
-    Editor.bridge = wrap(Editor.bridge, onBridge, 'around')
-    Editor.loadNote = loadNote20
+addHook("setupEditorShortcuts", onSetupShortcuts)
```

Comment:
Might be worthwhile to switch to `gui_hooks.editor_did_init_shortcuts`, while we're at it.